sys/linux/socket_netlink_generic_wireguard.txt

```diff
@@ -58,15 +58,21 @@ allowedip_policy_ipv6 {
 	WGALLOWEDIP_A_CIDR_MASK	nlattr[WGALLOWEDIP_A_CIDR_MASK, int8[0:128]]
 } [packed]
 
-# Limit number of keys to 5 to reduce search space.
-wireguard_key [
-	z	array[const[0, int8], NOISE_PUBLIC_KEY_LEN]
-	a	array[const[0xaa, int8], NOISE_PUBLIC_KEY_LEN]
-	b	array[const[0xbb, int8], NOISE_PUBLIC_KEY_LEN]
-	c	array[const[0xcc, int8], NOISE_PUBLIC_KEY_LEN]
-	d	array[const[0xdd, int8], NOISE_PUBLIC_KEY_LEN]
+wireguard_private_key [
+	zero	stringnoz["\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00"]
+	a	stringnoz["\xa0\x5c\xa8\x4f\x6c\x9c\x8e\x38\x53\xe2\xfd\x7a\x70\xae\x0f\xb2\x0f\xa1\x52\x60\x0c\xb0\x08\x45\x17\x4f\x08\x07\x6f\x8d\x78\x43"]
```

Comment:
Unfortunately this won't work, as `\` will get escaped. AFAIK currently the only way to encode arbitrary binary data is by using `int*` types, e.g. like [here](https://github.com/google/syzkaller/blob/master/sys/linux/vusb.txt#L935). You either need to do that or we need to extend syzlang to support such cases.

Reply:
In this case, I suppose I'd need int256. Does that exist?

Reply:
Seems like fixing the escaping might be an option?

Reply:
Alright, fixed this up. Please take a look.